### PR TITLE
Create Rule21-11C.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

File: docs/section21/Rule21-11C.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is not modeled with purchased heating: `if Rule-21-1 == "NOT APPLICABLE":`
+
```
Comment (mkarpman) [inline docs/section21/Rule21-11C.md:23]: =="UNDERTERMINED"

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

File: docs/section21/Rule21-11C.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+**Rule ID:** 21-11C  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled with a minimum turndown ratio of 0.25.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is not modeled with purchased heating.
```
Comment (mkarpman) [inline docs/section21/Rule21-11C.md:13]: Suggest to apply RDS irrespective of purchased heating

File: docs/section21/Rule21-11C.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
+  - Else, P-RMR is modeled with purchased cooling: `else:`
+
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
```
Comment (mkarpman) [inline docs/section21/Rule21-11C.md:44]: Same comment about not differentiating between projects with purchased and on-site cooling.

File: docs/section21/Rule21-11C.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    - Case 1: For heating hot water loop that boiler serves, if loop is modeled with a minimum turndown ratio of 25%: `if fluid_loop_b.heating_design_and_control.minimum_flow_fraction == 0.25: PASS`
+
+    - Case 2: Else, save component ID to output array for failed components:: `else: FAIL and failed_components_array.append(fluid_loop_b)`
+
+**Rule Assertion - RMR:**
+
+- Case 1: If all components pass: `if ALL_COMPONENTS == PASS: PASS`
+
+- Case 2: Else, list all failed components' ID: `else: FAIL and raise_message ${failed_components_array}`
+
```
Comment (mkarpman) [inline docs/section21/Rule21-11C.md:64]: Same question as for other RDSs about NOT APPLICABLE outcome and component vs global rule assertion.

File: docs/section21/Rule21-11C.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+**Function Call:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11.2, 12, 1a, 7a, 11.2a, 12a, continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-1", "SYS-5", "SYS-7", "SYS-11.2", "SYS-12", "SYS-1A", "SYS-7A", "SYS-11.2A", "SYS-12A"]): CHECK_RULE_LOGIC`
+
```
Comment (mkarpman) [inline docs/section21/Rule21-11C.md:27]: Please add systems that use purchased heat, same comments as 11-A and 11-B.

File: docs/section21/Rule21-11C.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+# Boiler - Rule 21-11C  
+
+**Rule ID:** 21-11C  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled with a minimum turndown ratio of 0.25.  
+**Rule Assertion:** B-RMR
```
Comment (supriyagoel) [inline docs/section21/Rule21-11C.md:6]: the rule description will also need to be updated to add purchased hot water

**Pull request merged**